### PR TITLE
Remove mixins

### PR DESCRIPTION
**Pull request by keepkeyjon**

### CODE REVIEW

File: packages/hdwallet-core/src/utils.ts [hunk 1]
```diff
@@ -138,3 +138,11 @@ export function satsFromStr (coins: string): number {
   let exponent = index > 0 ? 8 - (coins.length - index - 1) : 8
   return Number(coins.replace(/\./g, '')) * 10 ** exponent
 }
+
+export function applyMixins(derivedCtor: any, baseCtors: any[]): void {
```
Comment (ascension) [inline packages/hdwallet-core/src/utils.ts]: @keepkeyjon no longer needed?

File: packages/hdwallet-keepkey/src/ethereum.ts [hunk 2]
```diff
@@ -39,178 +39,168 @@ import {
...
 import * as EIP55 from 'eip55'
 
- /**
-  * Mixin Constructor that adds ETH support to a KeepKeyHDWallet
-  */
-export function KeepKeyETHWallet<TBase extends Constructor>(Base: TBase) {
-  return class KeepKeyETHWallet extends Base implements ETHWallet {
-    _supportsETH: boolean = true
-    transport: KeepKeyTransport
+export async function keepkey_ethSupportsNetwork (chain_id: number): Promise<boolean> {
```
Comment (ascension) [inline packages/hdwallet-keepkey/src/ethereum.ts]: @keepkeyjon can we drop the `keepkey_` prefix? When importing you can do `import { * as Something}` and `Something. ethSupportsNetwork`

File: packages/hdwallet-ledger/src/bitcoin.ts [hunk 3]
```diff
@@ -46,198 +47,185 @@ const segwitCoins = [
...
-
-      return {serializedTx:signedTx.payload,signatures:[]}
-    }
+  //sign createPaymentTransaction
+  let signedTx = await transport.call('Btc', 'createPaymentTransactionNew', inputs, paths, undefined, outputScriptHex, null, networksUtil[slip44].sigHash, segwit)
 
-    public async btcSupportsSecureTransfer (): Promise<boolean> {
-      return false
-    }
+  return {serializedTx:signedTx.payload,signatures:[]}
```
Comment (ascension) [inline packages/hdwallet-ledger/src/bitcoin.ts]: formatting, needs s p a c e s

File: packages/hdwallet-core/src/utils.ts (hunk 1 above)
Comment (keepkeyjon) [reply packages/hdwallet-core/src/utils.ts]: good call.

File: packages/hdwallet-keepkey/src/ethereum.ts (hunk 2 above)
Comment (keepkeyjon) [reply packages/hdwallet-keepkey/src/ethereum.ts]: sure

**Pull request merged**